ftd/image.ftd

```diff
@@ -259,3 +259,41 @@ dark: https://fastn.com/-/fastn.com/images/cs/show-cs-1-dark.jpg
 
 
 -- ftd.image-src just-light: https://fastn.com/-/fastn.com/images/cs/show-cs-1.jpg
+
+-- ds.h2: `fetch priority`
+
+Type: `optional` `string`
+
+Required: False
+
+The `fetchPriority` property allows you to signal high or low priority image fetches.
+This can be useful when applied to <img> elements to signal images that are "important" to the user experience early in the loading process.
+
+-- ds.rendered: Sample code using `fetch priority`
```

Comment:
Back tick is for "technical stuff", there is no such thing as `fetch priority` right, is that a thing? Is that the technical name of a property? A type? `fetch priory` is nothing. `fetch-priority` is a thing.